### PR TITLE
Add scale question to respond page

### LINKED ISSUE #111: Add scale questions to response

### PR DESCRIPTION
**Pull request by ajbalara**
# Description

Displays scale questions on the respond page. User can see the responses as a list of responses.
Future a PR will display the responses as a histogram on the summary page

## Issues

closes #111 

# How to test

Manual Test:
1. Go to localhost:8080
2. Create a new survey with 1+ scale questions and hit collect responses
3. Go to the respond page and confirm that the questions are there
4. Fill in the responses and click submit, repeat a couple times
5. Go to the summary page and confirm that the responses are displayed on the summary page

Integration tests:
* Run `mvn test`

# Checklist
**I've added tests**
- [x] Yes

- [ ] No

**I've updated the DB schema and UML diagram**
- [ ] Yes
- [x] No - Didn't change an models

### CODE REVIEW

**Comment by BraedenKloke** [comment]
@AliAbdollahian can you be responsible for reviewing this PR since it overlaps with the work you are doing?

File: src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java [hunk 1]
```diff
@@ -55,23 +55,30 @@ public String submit(@PathVariable("surveyID") int surveyID, @ModelAttribute Res
...
+                    // TODO: Implement
+                    continue;
+                } else if (question instanceof ScaleQuestion) {
+                    ScaleResponse scaleResponse = new ScaleResponse();
+                    Integer responseValue = response.get(i).getSelectedValue();
+                    System.out.println(responseValue);
+                    scaleResponse.setSelectedValue(responseValue);
+                    question.getResponses().add(scaleResponse);
+                    System.out.println(question.getResponses());
+                    System.out.println(question.getResponses().size());
```
Comment (JackieSL1) [inline src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java:75]: Should these print statements be here?

Comment (ajbalara) [reply src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java:75]: Ah good catch I forgot to remove them

**Review by AliAbdollahian — APPROVED** [review]
Looks good to me @ajbalara great job

**Pull request merged**